avaframe/ana4Stats/probAnaCfg.ini

```diff
@@ -20,6 +20,25 @@ peakLim = 1.0
 ##relTh = 0.75|0.9|0.8
 
 
+[PROBRUN]
+# settings for probabilistic run of computational module - add here parameters for parameter variation
+# on parameter is varied at a time - other parameters are kept at reference values
+# list of parameters that shall be varied - separated by |
+varParList = mu|relTh
+# provide reference values for these
+mu = 0.155
+relTh = 1.0
```

Comment:
I don't think having mu and relTh in here is a good idea. This might lead to very confusing settings, e.g. if a new default is introduced into the com1 cfg... 
And what happens if this is set to steps/range/multiple values?

Reply:
I changed this - now the parameter values in varParlist are read from com1DFACfg.ini or it's local - and error is thrown if there the value is not a single value but a variation entry